setup.cfg

```diff
@@ -32,6 +32,7 @@ python_requires = >=3.7
 setup_requires =
     setuptools_scm
 install_requires =
+    numpy
```

Comment:
Is it strictly necessary to make `numpy` a dependency? Obviously, no problem for astropy, but perhaps `doctestplus` is also used for non-numpy packages.

In principle, it would seem possible to avoid importing `numpy` until one was fairly certain one was dealing with a numpy ufunc (by which time a possible import error would be fine).

Anyway, not really a big deal - otherwise, this looks pretty good.

Reply:
We certainly could make it an optional dependency --- even an extra --- and import Numpy only if `--doctest-ufunc` is on. But thinking about packages that use pytest-doctestplus, I think that most of them also depend on Numpy, so it is not likely to be an extraneous dependency for most users.

Reply:
Yes, indeed, I think it is unlikely anybody would not use numpy as well. I'm fine with just using numpy and reconsidering if anyone speaks up...

Reply:
But given @bsipocz's comment, maybe just go the safer route!?

Reply:
Done.